Samples/BLEKeyboardSample/Program.cs

```diff
@@ -0,0 +1,47 @@
+using System;
+using System.Threading;
+
+using nanoFramework.Bluetooth.HID;
+using nanoFramework.Bluetooth.HID.Devices;
+
+namespace BLEKeyboardSample
+{
+	public class Program
+	{
+		private static Keyboard kbd;
+
+		public static void Main()
+		{
+			kbd = new Keyboard(deviceName: "nF BLE Keyboard",
+				deviceInfo: new DeviceInformation("nF", "BLEKBD1", "1", "01", "01", "01"),
+				protocolMode: ProtocolMode.Report,
+				plugAndPlayElements: new PnpElements(sig: 0x02, vid: 0xE502, pid: 0xA111, version: 0x210));
```

Comment:
I don't know if we should be making `sig`, `vid`, `pid`, etc.. be constants since they are meant to be unique to the product/device.
I used this [USB ID search site](https://the-sz.com/products/usbid/index.php?v=&p=&n=Keyboard) and couldn't find a `Generic Keyboard` entry.

The create pattern is a good idea for this but the only problem is what values to use for each type (keyboard, mouse, joystick, etc...).

Reply:
Then, maybe document it in the README so it's clear people can choose what they want or guive guidances or anything to help?

Reply:
Thanks @Ellerbach I have updated the README file and mentioned how these values are unique to the device/product and added a link to the database.